modules/combined/examples/geochem-porous_flow/forge/tests

```diff
@@ -14,6 +14,8 @@
     requirement = "The system shall be able to solve aqueous geochemical systems in contact with minerals, for water in a reservoir."
     issues = '#15693'
     design = 'forge.md'
+    # Geochemistry solver does not support recover
```

Comment:
Is there an issue for this?

Reply:
https://github.com/idaholab/moose/issues/15816